ci/test_python.sh

```diff
@@ -36,8 +40,7 @@ run_tests() {
 
   # Test with TCP/Sockets
   rapids-logger "TEST WITH TCP ONLY"
-  timeout 10m pytest --cache-clear -vs tests/
-  timeout 2m pytest --cache-clear -vs ucp/_libs/tests
+  ./ci/run_pytests.sh
 
   rapids-logger "Run local benchmark"
```

Comment:
Although they're technically "benchmarks" and not regular `pytest`s, I'd argue they're part of the test-suite and thus I would also move these into `run_pytests.sh`.

Reply:
That's fine with me. I wasn't sure if they were tests that should fail a build, or just verifying the benchmark code still runs. I'm happy to move them into the test script.

Reply:
I made a second `run_benchmark_pytests.sh` for these. That way if they cause issues (unrelated to correctness) in DLFW, I can choose not to run them.

Reply:
Those benchmarks should never fail either, they're distributed as part of the package because they're used by users to test interconnects, and thus they're part of the test suite here as we don't want them to be broken ever, so if they ever we would want that fixed just as any other test. We can keep `run_benchmark_pytests.sh` separate from `run_pytests.sh` if you wish but I would have combined them, when they're not combined an uninformed developer may think they're purely benchmarks in the sense of providing performance metrics and not part of the distributed package which should always be tested together.